apps/site/src/app/apply/sections/Form/BasicInformation.tsx

```diff
@@ -0,0 +1,110 @@
+"use client";
+
+import { useState } from 'react';
+import SingleSelect from "../Components/SingleSelect";
+import PasswordEye from "../Components/PasswordEye";
+import styles from "./Form.module.scss";
+
+//I'm not sure if we want to have different submission values
+//than display texts for the inputs, so I seperated both
+//for good measure
+
+const pronouns = [
+	{value: "she", text: "She/her/hers"},
+	{value: "he", text: "He/him/his"},
+    {value: "they", text: "They/them/theirs"},
+    {value: "ze", text: "Ze/zir/zirs"},
+    {value: "other", text: "Other:"},
+];
+
+const ethnicity = [
+	{value: "American", text: "American Indian or Alaskan"},
+	{value: "Asian", text: "Asian or Pacific Islander"},
+    {value: "Black", text: "Black or African American"},
+    {value: "Hispanic", text: "Hispanic"},
+    {value: "White", text: "White or Caucasian"},
+    {value: "Two-or-more", text: "Two or more races"},
+    {value: "Prefer not to answer", text: "Prefer not to answer"},
+    {value: "other", text: "Other:"}
+]
+
+export default function BasicInformation() {
+    //the password state here is supposed to be used for further validation when we
+    //check confirm password, but I'm pretty sure that will have to wait until we
+    //implement the submission process, so I just leave it here for now
+    const [password, setPassword] = useState("");
+
+    const [passwordVisible, setPasswordVisible] = useState(false);
+    const [confirmPasswordVisible, setConfirmPasswordVisible] = useState(false);
+
+    const handleChangePassword = () => {
+        setPasswordVisible(!passwordVisible)
+    }
+    
+    const handleChangeConfirmPassword = () => {
+        setConfirmPasswordVisible(!confirmPasswordVisible)
+    }
```

Comment:
Note: we won't need any of this as users will be required to log in before being able to view the application.

Reply:
huh, so would we even need a password at all?